src/fiat/bank-authorities.md

```diff
@@ -163,10 +163,10 @@ required together.
 | :----- | :-------------------------------------------- | :----------------------------------------------------------------- |
 | 403    | {% break _ DIRECT_DEBIT_ALREADY_AUTHORIZED %} | This bank authority cannot be changed as all fields have been set. |
 
-## Verify a bank authority
+## Verify a bank authority **DEPRECATED**
```

Comment:
deprecated features should point reader to alternative

Reply:
Link added